website/src/content/docs/general/FAQ.md

```diff
@@ -143,355 +98,45 @@ Don't use `$` in your passwords, it won't work due to an upstream [bug](https://
-
-[See our Support Policy](/
-
----
-
 ### cannot patch "APPNAME-redis" with kind StatefulSet
 
 `[EFAULT] Failed to update App: Error: UPGRADE FAILED: cannot patch "APPNAME-redis" with kind StatefulSet: StatefulSet.apps "APPNAME-redis" is invalid: spec: Forbidden: updates to statefulset spec for fields other than 'replicas', 'ordinals', 'template', 'updateStrategy', 'persistentVolumeClaimRetentionPolicy' and 'minReadySeconds' are forbidden`
 
 The solution: Check which apps have statefulsets by running:
 
 ```bash
-k3s kubectl get statefulsets -A | grep "ix-"
+kubectl get statefulsets -A
 ```
 
 Then, to delete the statefulset:
 
 ```bash
-k3s kubectl delete statefulset STATEFULSETNAME -n ix-APPNAME
+kubectl delete statefulset STATEFULSETNAME -n namespace
 ```
 
 Example:
 
 ```bash
-k3s kubectl delete statefulset nextcloud-redis -n ix-nextcloud
+kubectl delete statefulset nextcloud-redis -n nextcloud
 ```
 
 Once deleted you can attempt the update (or if you were already updated to latest versions, then edit and save without any changes).
 
 ## Operator-Related Errors
 
-### service "cnpg-webhook-service" not found
-
-`[EFAULT] Failed to update App: Error: UPGRADE FAILED: cannot patch "APPNAME-cnpg-main" with kind Cluster: Internal error occurred: failed calling webhook "mcluster.cnpg.io": failed to call webhook: Post "https://cnpg-webhook-service.ix-cloudnative-pg.svc/mutate-postgresql-cnpg-io-v1-cluster?timeout=10s": service "cnpg-webhook-service" not found`
-
-The solution:
-
-- Enter the following command
-
-```bash
-k3s kubectl delete deployment.apps/cloudnative-pg --namespace ix-cloudnative-pg
-```
-
-- Update `Cloudnative-PG` to the latest version, or if you already on the latest version, edit `cCloudnative-PG` and save/update it again without any changes.
-- If the app remains stopped, hit the start button in the UI for `Cloudnative-PG`.
-
-### "monitoring.coreos.com/v1" ensure CRDs are installed first
-
-`[EFAULT] Failed to update App: Error: UPGRADE FAILED: unable to build kubernetes objects from current release manifest: resource mapping not found for name: "APPNAME" namespace: "ix-APPNAME" from "": no matches for kind "PodMonitor" in version "monitoring.coreos.com/v1" ensure CRDs are installed first`
-
-The solution:
-
-- Install `Prometheus-Operator` first, then go back and install the app you were trying to install
-- If you see this error with Prometheus-Operator already installed, delete it and reinstall
-- While deleting Prometheus-{o}perator, if you encounter the error:
-
-`Error: [EFAULT] Unable to uninstall 'prometheus-operator' chart release: b'Error: failed to delete release: prometheus-operator\n'`
-
-Run the following command from the TrueNAS SCALE shell as root:
-
-```bash
-k3s kubectl delete namespace ix-prometheus-operator
-```
-
-Then install `Prometheus-Operator` again. It will fail on the first install attempt, but the second time it will work.
```

Comment:
Still applies, only will eventually point to kube-prometheus-stack instructions instead